code/__HELPERS/text.dm

```diff
@@ -263,9 +263,11 @@
 	return trim_left(trim_right(text))
 
 //Returns a string with the first element of the string capitalized.
-/proc/capitalize(t as text)
-	. = t[1]
-	return uppertext(.) + copytext(t, 1 + length(.))
+/proc/capitalize(t)
+	. = t

```

Comment:
Will this ever serve a purpose?

Reply:
Isn't that what's used to auto-cap the first letter of sentences when spoken IC?

Reply:
`""` vs `null`, i didn't go over the uses to figure out if any cares so just passed it on.

Reply:
Huh, fair enough